bundles/com.espressif.idf.core/src/com/espressif/idf/core/ExecutableFinder.java

```diff
@@ -1,106 +1,12 @@
+/*******************************************************************************
+ * Copyright 2023 Espressif Systems (Shanghai) PTE LTD. All rights reserved.
+ * Use is subject to license terms.
+ *******************************************************************************/
 package com.espressif.idf.core;
 
-import java.io.File;
-import java.lang.reflect.Method;
-
-import org.eclipse.core.filesystem.EFS;
-import org.eclipse.core.filesystem.IFileStore;
 import org.eclipse.core.runtime.IPath;
-import org.eclipse.core.runtime.Path;
-import org.eclipse.core.runtime.Platform;
-
-import com.espressif.idf.core.util.StringUtil;
 
-public class ExecutableFinder
+public interface ExecutableFinder
```

Comment:
Would it be better to rename the interface to `IExecutableFinder` for better readability

Reply:
"I" prefix is not a standard naming convention in Java for interfaces. Java standard interfaces as a reference:
Map, List, Collection, etc.